app/assets/javascripts/components/Filters.js

```diff
@@ -43,5 +55,53 @@ define(['jquery', 'DoughBaseComponent'], function($, DoughBaseComponent) {
     this.$form.submit();
   }
 
+  /**
+  * Assign/Remove listner on toggle
+  */
+  Filter.prototype._toggleFilters = function(self, isLoad) {  
+    if ( $(window).width() < this.breakpoint ) {
+      self._toggleMobile(self, isLoad);
+    } else {
+      self._toggleDesktop(self, isLoad);
+    }
+    self.windowSize = $(window).width();
+  } 
+
+  /**
+  * Is Mobile
+  *   was previously desktop, or page just loaded, so rest for mobile
+  */
+  Filter.prototype._toggleMobile = function(self, isLoad) {
+    if ( self.windowSize >this.breakpoint || isLoad ) {
+      self.$form.removeClass(this.activeClass);
+      self.$wrap.slideUp(0);
+
+      self.$toggle.on('click', function(e) {
+        e.preventDefault();
+        self._toggleAnimate();
+      })
+    }
+  }
+  /**
+  * Is Desktop
+  *   was previously mobile, or page just loaded, so rest for desktop
+  */
+  Filter.prototype._toggleDesktop = function(self, isLoad) {
+    if ( self.windowSize < this.breakpoint || isLoad ) {
```

Comment:
If I understood correctly, this verification is duplicated. Can this verification be a function? Wdyt?

Reply:
There is a similar if statement on line 75 but they are not the same

Reply:
Scrap that, I've put the first part of the if statement in a function